### ISSUE TITLE
SQL queries inside f-strings (Python 3.6+) are not being checked

### ISSUE THREAD

**Issue by fgimian** [pgjones/flake8-sql#8]
Hey there, another little issue I discovered is that SQL inside f-strings are not being checked:

e.g.

Lines 61 - 65:

```python
        for item_id, title, uuid, flags in self.conn.execute(f'''
            SELECT {table}.item_id, {table}.title, items.uuid, items.flags
            FROM {table}
            JOIN items ON items.rowid = {table}.item_id
        '''):
```

These show no errors, but if I remove the `f` prior to the string starting, I see:

```
./elite/libraries/launchpad_builder.py:65:0: Q447 root_keywords SELECT and FROM are not right aligned
./elite/libraries/launchpad_builder.py:65:0: Q449 token { should be aligned to the right of the river
./elite/libraries/launchpad_builder.py:65:0: Q449 token table should be aligned to the right of the river
./elite/libraries/launchpad_builder.py:65:0: Q447 root_keywords SELECT and JOIN are not right aligned
./elite/libraries/launchpad_builder.py:65:0: Q449 token items should be aligned to the right of the river
```

Kindest Regards
Fotis

**Comment by pgjones** [comment]
I think I've fixed this in 6da30ca05c2a0d9fc842f53ce2317172c6dba450, could you try it and see if it fixes your issue(s)?

**Comment by fgimian** [comment]
Thanks so much @pgjones, I'm so sorry for the delay, this week has been a bit nuts.  I'll try to check this out on the upcoming weekend and let you know if it's all good 😄 

Cheers
Fotis

**Comment by fgimian** [comment]
I can confirm that the fix implemented indeed repairs this issue, thank you so much! 😄 

Cheers
Fotis

**Comment by pgjones** [comment]
Thanks, I've released 0.3.0
